masterlist.yaml

```
@@ -3473,6 +3473,27 @@ plugins:
       - crc: 0x92D9C086
         util: 'TES4Edit v4.0.4b'
 
+  - name: 'ShinySeptims_(Septimus|Trajanus)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/23733/'
+        name: 'Shiny Septims - Oblivion'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Shiny Septims - Oblivion ESP' ]
+        condition: 'many("ShinySeptims_(Septimus|Trajanus)?\.esp")'
```

There is no plugin available on the page named *ShinySeptims_.esp*, so the `?` quantifier is unnecessary.

```suggestion
  - name: 'ShinySeptims_(Septimus|Trajanus)\.esp'
    url:
      - link: 'https://www.nexusmods.com/oblivion/mods/23733/'
        name: 'Shiny Septims - Oblivion'
    msg:
      - <<: *useOnlyOneX
        subs: [ 'Shiny Septims - Oblivion ESP' ]
        condition: 'many("ShinySeptims_(Septimus|Trajanus)\.esp")'
```